dashboard/test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -786,6 +786,49 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
     assert_equal auth[:credentials][:refresh_token], partial_user.oauth_refresh_token
   end
 
+  test 'google_oauth2: redirects to maker/display_google_oauth_code when google login is successful if the session[:user_return_to] is set to /maker/display_google_oauth_code' do
```

Comment:
nit: these test descriptions are pretty long. consider simplifying to something like (just brainstorming) 'google_oauth2: redirects to maker display code if present as user_return_to on successful google login'